src/test/resources/goal-output/plume-lib-1.1.0/plume.WeakIdentityHashMap_goal.json

```diff
@@ -362,7 +362,7 @@
     "returnTag": {
       "comment": "the value to which this map maps the specified key, or null if the map contains no mapping for this key.",
       "kind": "RETURN",
-      "condition": ""
+      "condition": "target.containsKey(args[0])?result==null"
```

Comment:
This looks wrong to me. Shouldn't be: `target.containsKey(args[0])==false ? result==null`. Another possibility is:
`target.containsKey(args[0]) ? result!=null : result==null`.

Reply:
Right - fixed